CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+Before posting a new bug, please ensure that there is no previous issue
+addressing the same problem.
+To do that, go on the [issues](https://github.com/oncodash/oncodash/issues?q=is%3Aissue)
+tab of the project and search for all (open or closed) issues.
+
+When filing a report, make sure to answer these five questions:
+
+- What version of Oncodash are you using?
+    - If you use a release, indicate its tag, else indicate the specific commit hash. 
+- What operating system are you using?
+- What versions of Docker, Python and Typescript?
+- What did you do?
+    - Explain the steps to reproduce the bug from a fresh start.
+- What did you expect to see?
+- What did you see instead?
+
+
+How to Submit Code
+==================
+
+1. Create your own fork of the project.
+2. Checkout a branch named after the following convention:
+    - `<type>[#:ID]_[scope/]<three-keywords-max>`
+    - `type` may be: `feat`, `fix`, `doc`, `refactor`, `build`, `ci`, etc.
+    - `ID` (optional) refers to the issue number.
+    - `scope` is a generic category, like "frontend", "backend" or "widget".
+    - Examples: `feat_widget/timeline` or `fix#12_backend/import-clinical`
+3. Commit your changes using the [Conventional Commits](https://www.conventionalcommits.org/) convention:
+    - First line: `<type>[(scope)][!]: <short description>`,
+      the exclamation mark indicating a breaking change (usually in an API).
+    - Empty line, followed by a multi-line long description.
+    - Optional footer as `Ref:#<ID>` on separate line(s), to [reference a related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+      or explain the breaking change.
+    - Example:
+        ```
+        fix(API)!: use dictionary instead of list in inference server
+
+        This should improve performances by allowing queries in O(log n) instead of O(n).
+
+        Ref:#123
+        BREAKING CHANGE: expects a key-value pair instead of single items.
+        ```
+4. Ensure that you did not introduce bugs by adding unit tests that run your new code.
+5. Ensure that you did not introduced a regression by running all the tests
+   yourself, from within the project's containers.
+6. Add explanations about your new feature in the documentation.
+7. Push on your own repository and click on the "Create pull request" green
+   button that should appear on your fork's page (or go to the original
+   project page and click on "[Pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)").
+8. In the "base" dropdown menu, select the "dev" branch.
+
+
+How to Merge code
+=================
+
+Our policiy for merging pull requests is to squash all commits into one.
+If your pull request consists in several commits, we will squash them manually,
+but the commit message may be cleaner if you rebase-and-squash your own branch before
```

Comment:
I think it's better to let GitHub handle squashing so that the individual commits can be recovered if needed. Rebasing, and in extreme cases cherry picking, is a good practice before pushing to a branch that will end up in a PR.